### PR TITLE
[MINI-3241] Add Points Settings

### PR DESCRIPTION
**Pull request by laubengaier**
# Description

Adding points to settings to change the points values 

Settings             |  Edit Points             | Points fetched
:-------------------------:|:-------------------------:|:-------------------------:
 |  | 

## Links
https://jira.rakuten-it.com/jira/browse/MINI-3241

# Checklist
- [x] I have read the [contributing guidelines](CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

File: Example/Controllers/PointsSettingsTableViewController.swift [hunk 1]
```diff
@@ -0,0 +1,61 @@
+//
+//  PointsSettingsTableViewController.swift
+//  MiniApp_Example
+//
+//  Created by Laubengaier, Timotheus | MTSD on 2021/07/16.
+//  Copyright © 2021 Rakuten Group, Inc. All rights reserved.
+//
```
Comment (rleojoseph) [inline Example/Controllers/PointsSettingsTableViewController.swift:7]: Please remove the header comment

**Review by rleojoseph — CHANGES_REQUESTED** [review]
Please update the Changelog as well.

File: Example/Controllers/PointsSettingsTableViewController.swift (hunk 1 above)
Comment (laubengaier) [reply Example/Controllers/PointsSettingsTableViewController.swift]: oh yeah, I just removed the header comment and added a changelog feature entry

**Review by corycaywood — COMMENTED** [review]
Looks good 👍 

I think the Points feature isn't documented in the Userguide yet? Could you also add that documentation?

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,9 @@
 **Sample app**
 
 - **Feature:** Display of a banner on message contact picker when required
+- **Feature:** Added Points Interface `getPoints` to retrieve Rakuten Points
+- **Feature:** Added `rakuten.miniapp.user.POINTS` custom permission
```
Comment (corycaywood) [inline CHANGELOG.md:13]: These two items should go under the "SDK" section above.

Comment (laubengaier) [reply CHANGELOG.md:13]: yeah true just changed it 👍

**Comment by laubengaier** [comment]
> Looks good 👍
> 
> I think the Points feature isn't documented in the Userguide yet? Could you also add that documentation?

@corycaywood yeah sure, I've updated it. is that ok as it is?

**Review by corycaywood — APPROVED** [review]
Looks good 👍

**Review by rleojoseph — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**